tensorrt_llm/_torch/auto_deploy/distributed/common.py

```
@@ -189,65 +181,57 @@ def initialize(
+                    f"Checking port for: {lib=}, {local_rank=}, {world_size=}, {port=} (attempt {attempt + 1})"
+                )
+                if _is_port_available(port):
+                    port_ok = True
+                    break
+                ad_logger.warning(f"Port {port} already in use, will retry with new port")
+                port = get_free_port()
+            else:
+                init_error = RuntimeError(
+                    f"Failed to find available port after {max_retries} attempts"
+                )
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)